week-2-practise/level-1/6-map.js

```diff
@@ -11,7 +11,7 @@ function tidyUpString(str) {
 }
 
 var mentors = ["/Daniel ", "irina ", " Gordon", "ashleigh "];
-var mentorsTidy; // ONLY EDIT THIS LINE
+var mentorsTidy = str.map (tidyUpString); // ONLY EDIT THIS LINE
```

Comment:
What's `str` here? You need to call `map` on the array and it will apply the function to each member.